### PR TITLE
Feature: Add onboarding screens

### PR DESCRIPTION
**Pull request by kibongos40**
# Description

This pull request implements a user onboarding flow to guide new users through the app. It introduces onboarding screens. The PR also includes updated splash screen, logo and favicon.

We would appreciate your feedback and review.

image
image

### CODE REVIEW

File: app/onboarding/assets/fonts/urbanist.ttf
Comment (chrysloi) [inline app/onboarding/assets/fonts/urbanist.ttf:1]: this file shouldn't be here. consider using the fonts that were set in the tailwind.config file.

**Review by chrysloi — COMMENTED** [review]
Please do move these assets to the asset folder on the root!

**Comment by fredshema** [comment]
Also squash your commits into one

**Comment by justinmihigo** [comment]
Consider using the PR description templates your description is not good looking

File: app/onboarding/assets/fonts/urbanist.ttf
Comment (kibongos40) [reply app/onboarding/assets/fonts/urbanist.ttf:1]: I'm working on it.

**Comment by kibongos40** [comment]
I am working on all requested changes.

**Comment by kibongos40** [comment]
Will reopen after fixing all changes.

**Pull request closed without merging**